examples/pyplots/pyplot_lorder.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as pyplot
```

Comment:
While I prefer to spell out pyplot as you've done, you should do`import matplotlib.pyplot as plt` to be consistent with our other examples.